code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -308,6 +308,7 @@
 	var/eggs_left = 0
 	var/eggs_added_from_eating = rand(1, 4)
 	var/max_eggs_held = 8
+	var/datum/callback/lay_egg_callback = CALLBACK(src, .proc/egg_laid)
```

Comment:
you could've just done this in place but whatever

Reply:
i should move it